### ISSUE TITLE
Unable to use UserNotifications' framework with Theos app in a jailbroken environment

### ISSUE THREAD

**Issue by Luki120** [theos/theos#661]
### What are the steps to reproduce this issue?

1) Make a theos app, either Objective-C or Swift, doesn't matter.
2) Paste this in a `viewDidLoad` method or the equivalent:
```swift
UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .badge, .sound]) { _, _ in }
```

### What happens?

Nothing

### What were you expecting to happen?

For the app to display a stock iOS alert that asks the user wether they'd like to give access to the app to send them notifications

### Additional context

This same code works in the same app if it's compiled as an ipa and sideloaded. My guess is that since the sideloaded app shows at the bottom of the settings app like if it was a regular app, that's what makes it work since iOS thinks it's a stock app and treats it like so.

### What OS are you seeing the problem on?

Linux/WSL

### What OS version does it have installed?

Kubuntu 21.04

### What toolchain and version are you using?

Swift toolchain 5.3.2

### Which SDK version are you using?

iOS 14.5

### What OS is your client device running?

iOS

### What OS version does it have installed?

iOS 14.5.1

### Which device model is it?

iPod touch 7th generation

### Which jailbreak is it using (if relevant)?

Checkra1n

**Comment by opa334** [comment]
Add SBAppUsesLocalNotifications true to your Info.plist, then notifications will work

**Comment by Luki120** [comment]
@opa334 like this?
```xml
<key>SBAppUsesLocalNotifications</key>
<true/>
```
I'm not getting the stock alert

**Comment by opa334** [comment]
Yes, there will be no stock alert, notifications will just work (and they can be configured inside Settings -> Privacy too)

**Comment by Luki120** [comment]
I've tried this code but there's no notification

```swift
func postNotification() {
	let content = UNMutableNotificationContent()

	content.title = "Cathal"
	content.body = "Well done! You've completed a Pomodoro session. Starting break now.."
	content.sound = UNNotificationSound.default

	let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 5, repeats: false)
	let request = UNNotificationRequest(identifier: "pomodoroDone", content: content, trigger: trigger)

	UNUserNotificationCenter.current().add(request, withCompletionHandler: nil)
}
```
I'm calling it in `.onAppear`, which is like a `viewWillAppear:` from UIKit. Also I'm not seeing my app on Settings -> Privacy, or what should I be looking for there?

**Comment by opa334** [comment]
my bad, settings -> notifications is what you want to look at, also you still have to request authorization for notifications at least once before sending one, this won't show a permission alert tho because it's a system app

**Comment by opa334** [comment]
after the SHELL variable in common.mk is set, getting the working directory breaks

![Bildschirmfoto 2022-09-19 um 17 16 15](https://user-images.githubusercontent.com/18235760/191052440-f24a8a0a-c7e8-4933-9921-01dce286c100.png)

![Bildschirmfoto 2022-09-19 um 17 16 59](https://user-images.githubusercontent.com/18235760/191052386-57c65a50-fa4a-4de8-85ec-3d6d06440d83.png)

**Comment by Luki120** [comment]
@opa334, Still no notifs with this code :/
```swift
.onAppear {
	timerViewModel.requestAuthorization()
	DispatchQueue.main.asyncAfter(deadline: .now() + 4) {
		timerViewModel.postNotification()
	}
}
```
But the app ain't showing up at all in that menu either, so ig the problem is something else. If you want to try it out here's the [app](https://github.com/Luki120/iOS-Apps/tree/main/Basics/Cathal)
